content/source/docs/enterprise/private/vmware-setup-guide.html.md

```diff
@@ -92,14 +91,16 @@ the internal database or vault may result in serious performance issues.
 
 ### Object Storage (S3)
 
-An [S3 Standard](https://aws.amazon.com/s3/storage-classes/) bucket must be
+An [S3 Standard](https://aws.amazon.com/s3/storage-classes/) bucket, or compatible storage, must be
 specified during the PTFE installation for application data to be stored
 securely and redundantly away from the virtual servers running the PTFE
-application. This S3 bucket must be access via the network to the PTFE virtual
+application. This object storage must be accessible via the network to the PTFE virtual
 machine as well as the postgres server. Vault is used to encrypt all
-application data stored in the S3 bucket. This allows for further [server-side
+application data stored in this location. This allows for further [server-side
 encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/serv-side-encryption.html)
-by S3 if required by your security policy.
+by S3 if required by your security policy. 
+
+S3-compatible storage can be Google Cloud storage, Azure blob storage, or [ceph](https://ceph.com/), among many others. Please feel free to reach out to [support](https://www.hashicorp.com/support) with questions. 
```

Comment:
This is somewhat incorrect.  We have first-class support for S3, GCS, and Azure's storage, or you can use an S3-_compatible_ service such as ceph.  If you can use the first-class storage service support, you should.  In fact, Google's S3-compatibility is pretty marginal, from what I've heard.

Reply:
Maybe I could alter that to read "Supported storage options include AWS S3, Azure, Google Cloud Storage, and ceph, among others." ?

Reply:
Yes, but I'm hesitant to say we "support" ceph.  I'm not entirely sure we should remove the minio reference, as well.  The lack of versioning in minio is important, but I'm not really comfortable implying ceph is the only solution.  I'm also not sure we call out the need (well, desire) for versioning outside of the S3 bucket setup…